test/e2e/scheduling/preemption.go

```diff
@@ -122,47 +122,50 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		var podRes v1.ResourceList
 
 		// Create one pod per node that uses a lot of the node's resources.
-		ginkgo.By("Create pods that use 2/3 of node resources.")
+		ginkgo.By("Create pods that use 4/5 of node resources.")
 		pods := make([]*v1.Pod, 0, len(nodeList.Items))
 		// Now create victim pods on each of the node with lower priority
```

Comment:
This should say something like:

Create pods in the cluster. One of them has low priority, making it the victim for preemption.

Reply:
Updated